sycl/test/lit.cfg.py

```diff
@@ -159,6 +159,9 @@
 if config.native_cpu_be == "ON":
     config.available_features.add("native_cpu_be")
 
+if config.native_cpu_ock == "ON":
+    config.available_features.add("native_cpu_ock")
```

Comment:
Is oneAPI Construction Kit really tied to the native cpu? Or maybe we should remove the "native_cpu" prefix here (and expand the ock abbreviation slightly)?

Reply:
Currently it gets fetched only when building the native cpu backend, this prefix is intended to be "native cpu with OCK"

Reply:
Ok, we can always change it later it other uses are added.